lib/open-amp/resource_table.h

```diff
@@ -17,8 +17,17 @@ extern "C" {
 #if (CONFIG_OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF > 0)
 
 #define VDEV_ID                 0xFF
+#ifdef CONFIG_IPM_RX_CHANNEL_ID
+#define VRING0_ID CONFIG_IPM_RX_CHANNEL_ID
+#else
```

Comment:
I have a doubt....
As CONFIG_IPM_RX_CHANNEL_ID has a default value set to 0  do we need the `#ifdef CONFIG_IPM_RX_CHANNEL_ID`
or just  `#define VRING0_ID CONFIG_IPM_RX_CHANNEL_ID` is working in all cases?